saturn-r/postBuild

```diff
@@ -0,0 +1,49 @@
+# https://cran.rstudio.com/bin/linux/debian/
+sudo apt-key adv --keyserver keys.gnupg.net --recv-key 'E19F5F87128899B192B1A2C2AD5F960A256A04AF'
+
+sudo apt-get update
+sudo apt-get install -y \
+    --no-install-recommends \
+        software-properties-common
+
+sudo su -c \
+    "echo 'deb http://cloud.r-project.org/bin/linux/debian buster-cran40/' >>/etc/apt/sources.list"
+
+sudo apt update
+sudo apt install -y -t buster-cran40 \
+    r-base \
+    r-base-dev 
+
+sudo apt-get update
+sudo apt-get install -y \
+    jupyter-client \
+    devscripts \
+    libxml2 \
+    libxml2-dev \
+    libcurl4-openssl-dev \
+    libssl-dev \
+    libzmq3-dev \
+    qpdf
+
+# make it possible to `install.packages()` - write permissions required.
+sudo chown 1000:1000 -R $R_LIBS
+sudo chmod 777 -R $R_LIBS
+
+# install other packages
+sudo Rscript -e "install.packages(c( \
+        'caret', \
+        'data.table', \
+        'dplyr', \
+        'ggplot2', \
+        'IRkernel', \
+        'lightgbm', \
+        'Rcpp', \
+        'readr', \
+        'remotes', \
+        'reticulate', \
+        'stringr', \
+        'tidyr', \
+        'xgboost' \
+    ), Ncpus = 6, \
```

Comment:
I'm not sure that you can rely on the node used in our CI having this many cores. I believe you'll get a single 2-core CPU: https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources

```suggestion
    ), Ncpus = max(c(1, parallel::detectCores() - 1)), \
```